packages/piro/src/Piro_TempusSolver.hpp

```diff
@@ -212,6 +212,10 @@ class TempusSolver
   //from Albany.  
   bool abort_on_fail_at_min_dt_;
 
+  int sens_method_; 
+  //sens_method = 0: No sensitivities
+  //sens_method = 1: Forward sensitivities
+  //sens_method = 2: Adjoint sensitivities 
```

Comment:
why not use an enum here instead of int?

Reply:
I agree - I have switched the code to use enum for this variable.